lib/test_ocr.py

```diff
@@ -0,0 +1,98 @@
+# Copyright 2018 The Fuego Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+
+
+import sys
+import os
+fuegoRoot = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+sys.path.insert(0, os.path.join(fuegoRoot, 'lib'))
+sys.path.insert(0, fuegoRoot)
+import settings
+settings.fuegoRoot = fuegoRoot
+import numpy as np
+import logging
+import OCR
+
+assert os.path.isfile('./test_OCR1.jpg')
+assert os.path.isfile('./test_OCR2.jpg')
+
+def test_load_image():
+    #load_image( infilename )
+    testdata = OCR.load_image('test_OCR1.jpg')
+    assert type(metadatastrip) == type(np.array([]))
+
+def test_save_image():
```

Comment:
It would be better to use pytest support for temporary files: https://docs.pytest.org/en/latest/tmpdir.html